### DISCUSSION TITLE
Can't read Air temperature and humidity on Remotec's zxt800

### DISCUSSION THREAD

**Discussion by MickaelC26** [zwave-js/zwave-js#6599] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

Hello,

**What is happening?**
After the inclusion of my zxt800, I can see the endpoint 49-1-Air temperature (but not humidity), but it doesn't seem to refresh by himself unless I click on refresh (in zwave JS UI). 

However, I'm using a zwave sniffer in parralel and can see that when I blow on the device, both temperature and humidity is being sent (see screenshot). I also tried this device with an everspring zwave developpement usb dongle, and it works correctly. 

**What did you expect to happen instead?**
Both Humidity and temperature being correctly displayed and interpreted

I guess it come from my gateway only

Anything else we should know?

I saw this issue https://github.com/zwave-js/node-zwave-js/issues/1181 where it seem related but the answer is a fix on a more recent version of zwaveJS, but I would prefer not update my current version. 
It would be perfect if I could maybe redirect or force those two endpoint using a custom json file for the zxt800

### Software versions

Driver (node-zwave-js): 10.10.0

Z-Wave JS UI: 8.8.5

### Device information

Manufacturer: Remotec
Model name: zxt800
Node ID: 7

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwave-js-ui-store (4).zip](https://github.com/zwave-js/node-zwave-js/files/13875687/zwave-js-ui-store.4.zip)
![Capture d'écran 2024-01-09 154719](https://github.com/zwave-js/node-zwave-js/assets/123379128/00c8ca1d-9b06-467e-8c9d-99f357a9f336)
[zxt800.json](https://github.com/zwave-js/node-zwave-js/files/13875694/zxt800.json)

Find attached the log where I include the device, then I force the temperature and humidity to be sent (by blowing on it), and then after sending a refresh from zwave js ui.

**Comment by AlCalzone** [comment]
You may have to update to at least v10.18.0, which includes #5747. In 10.10, this endpoint may be incorrectly considered to be unnecessary and therefore hidden/ignored.

**Reply by AlCalzone** [reply]
If that's not an option, maybe adding the `compat` flag `"preserveEndpoints": [1, 2]` (not sure which ones you need) to the configuration file and re-interviewing may help.

**Reply by MickaelC26** [reply, marked as answer]
I couldn't make` preserveEndpoints` nor` removeEndpoints` works (strange because I already made it by the past for an other device).
But `preserveRootApplicationCCValueIDs` to true duplicated me the endpoint 0 and it works now.
Thanks for you help !
